### ISSUE TITLE
AVX2 for ppv-lite86

### ISSUE THREAD

**Issue by kazcw** [cryptocorrosion/cryptocorrosion#11]

**Comment by vks** [comment]
Also see #17.

**Comment by kazcw** [comment]
Right, status update: implemented for u32x4x4 in #17; the rest of the trait matrix uses the SSE2 code for now (but compiles to vex-encoded versions, aka "AVX128")
